File: code/1007/1007.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const num1 = parseInt(lines[0]);
const num2 = parseInt(lines[1]);
const num3 = parseInt(lines[2]);
const num4 = parseInt(lines[3]);

console.log(`DIFERENCA = ${(num1 * num2) - (num3 * num4)}`);

File: code/1007/1007.cs
using System;

class URI {
  static void Main(string[] args) {
    int num1, num2, num3, num4;
    int.TryParse(Console.ReadLine(), out num1);
    int.TryParse(Console.ReadLine(), out num2);
    int.TryParse(Console.ReadLine(), out num3);
    int.TryParse(Console.ReadLine(), out num4);
    int diff = (num1 * num2) - (num3 * num4);
    Console.WriteLine("DIFERENCA = " + diff);
  }
}